### DISCUSSION TITLE
Cant launch app after updating to .net8

### DISCUSSION THREAD

**Discussion by Petrarca181** [dotnet/maui#16230] [Q&A] [answered]
Hello I updated my existing app to .net 8 preview 6, but can't make it work. 

`**System.MissingMethodException:** 'Method not found: Microsoft.Maui.Graphics.Size Microsoft.Maui.IContentView.CrossPlatformMeasure(double,double)'`

I get this at app startup.

**Comment by Petrarca181** [comment, marked as answer]
Problem was https://github.com/PureWeen/ShanedlerSamples

**Comment by dgerding** [comment]
@Petrarca181 Any chance you understand if they intention is to fork these in to main Maui later?

**Reply by wallym** [reply]
I've just tried to have a .net 8 maui app with the syncfusion image editor control and boom.  I get this error.  While I'm sure that I can jump back to .net 7, i'm the dangerous type.  Any idea how to fix the problem right now?

**Comment by dgerding** [comment]
Anyone else seeing this behavior again? 

I added Pureween "shims" and regrettably, they didn't do the trick.

System.MissingMethodException: 'Method not found: 'Microsoft.Maui.Graphics.Size Microsoft.Maui.IContentView.CrossPlatformMeasure(Double, Double)'.'

Any ideas appreciated.

**Reply by OmidTavana** [reply]
same for me when im using Telerik RadAutoComplete
